<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCalibrationTaskClass</class>
 <widget class="QMainWindow" name="CameraCalibrationTaskClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraCalibrationTask</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="oldPic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>原始画面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="newPic">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>矫正画面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="openCameraBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrationBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showUndistortedBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>显示矫正画面</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CameraCalibrationTask.qrc"/>
 </resources>
 <connections/>
</ui>
